meilisearch-http/src/main.rs

```diff
@@ -94,37 +93,41 @@ pub fn print_launch_resume(opt: &Opt, data: &Data) {
 888       888  "Y8888  888 888 888  "Y8888P"   "Y8888  "Y888888 888     "Y8888P 888  888
 "#;
 
-    info!("{}", ascii_name);
+    eprintln!("{}", ascii_name);
 
-    info!("Database path: {:?}", opt.db_path);
-    info!("Start server on: {:?}", opt.http_addr);
-    info!("Environment: {:?}", opt.env);
-    info!("Commit SHA: {:?}", env!("VERGEN_SHA").to_string());
-    info!(
-        "Build date: {:?}",
+    eprintln!("Database path:\t\t{:?}", opt.db_path);
+    eprintln!("Server listening on:\t{:?}", opt.http_addr);
+    eprintln!("Environment:\t\t{:?}", opt.env);
+    eprintln!("Commit SHA:\t\t{:?}", env!("VERGEN_SHA").to_string());
+    eprintln!(
+        "Build date:\t\t{:?}",
         env!("VERGEN_BUILD_TIMESTAMP").to_string()
     );
-    info!(
-        "Package version: {:?}",
+    eprintln!(
+        "Package version:\t{:?}",
         env!("CARGO_PKG_VERSION").to_string()
     );
 
+    eprintln!();
+
     if let Some(master_key) = &data.api_keys.master {
```

Comment:
This might not be pertinent in this PR, but isn't it unsafe to put the Master key (and maybe private/public keys) in the logs?

Reply:
@Kerollmops

Reply:
Not sure why @qdequele have done this, but yeah it can be unsafe if any user have access to the logs of the running server.

Reply:
Should I create an issue on this?

Reply:
I can fix this here, seems related enough

Reply:
I tried to create a suggestion, but it gets messy as it is done over a mix of removed and added lines. The main idea is just to tell the user that a Master key was set, and that he needs authentication for requests, but deleting all the logs with sensitive keys inside ;)